javascript/javascript1/week3/NOnoN0nOYes(Note_taking_app).js

```diff
@@ -0,0 +1,47 @@
+// ----------------------------------------------------------------------------------NOnoN0nOYes (Note taking app)\
+// ----------------------------------------------------------------------------------Save a note
+
+const notes = [];
+
+function saveNote(content, id) {
+  notes.push({content:content, id:id});
+}
+
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// --------------------------------------------------------------------------------------------Get a note
+
+function getNote(id) {
+  // your code here
+  for (let i = 0; i < notes.length; i++){
+    if (id === notes[i].id) {
+       return notes[i];
+    } else if (isNaN(id) || id === null) {
```

Comment:
it makes sense to check if `isNan(id)` or `id === null` before the `for` loop. 
Motivation is following: it makes sense to check that input arguments are correct before trying to anything at all.